doc/source/management/data_at_rest_encryption.rst

```diff
@@ -477,16 +461,81 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. rubric:: Redo log encryption
+
+Implemented in 8.0.16-7, the key rotation for the redo log has been redesigned to rotate the key with `SELECT rotate_system_key("percona_redo").
+
+Implemented in 8.0.16-7, the only supported values for `set global innodb_redo_log_encrypt` are the following:
 
```

Comment:
Implemented in :rn:`8.0.16-7`. 

Supported values for :variable:`innodb_redo_log_encrypt` are the following:

Reply:
Updated.